src/map/prepareLayer.js

```diff
@@ -270,7 +271,11 @@ function readData(mapId, layer, dispatch, doUpdateTsLayer) {
     if (layerObj.aggregate && layerObj.aggregate.filter) {
       layerObj.filterOptions = generateFilterOptions(layerObj);
     }
-    renderData(mapId, layerObj, dispatch, doUpdateTsLayer);
+    const geojsonErrors = hint(sourceURL.data).filter(e => e.level !== 'message');
+
+    if (!geojsonErrors || !geojsonErrors.length) {
+      renderData(mapId, layerObj, dispatch, doUpdateTsLayer);
+    }
```

Comment:
This pr looks good. I'm thinking it would be awesome to have an error message shown when geojsonErrors.length is true

Reply:
@kahummer do we currently have a way to show such error messages?

Reply:
@cKellyDesign how about logging the message on the console just to inform better what part of the geometry isn't correct. an else that logs the ``` geojsonErrors["message"] ```

Reply:
ok maybe we can do a `console.warn(..)` then